ocs_ci/ocs/ocp.py

```diff
@@ -696,3 +697,69 @@ def rsync(src, dst, node, dst_node=True, extra_params=""):
         except CommandFailed:
             log.warning(f"Pod {pod_name} wasn't successfully deleted!")
             raise
+
+
+def get_images(data, images=None):
+    """
+    Get the images from the ocp object like pod, CSV and so on.
+
+    Args:
+        data (dict): Yaml data from the object.
+        images (dict): Dict where to put the images (doesn't have to be set!).
+
+    Returns:
+        dict: Images dict like: {'image_name': 'image.url.to:tag', ...}
+    """
+    if images is None:
+        images = dict()
+    data_type = type(data)
+    if data_type not in (dict, list):
```

Comment:
No need for those 2 lines.
Line 718 will check if dict and line 734 will check if list. 
If both not equal, return images like in line 717

Reply:
Can we add warning to logs if users didn't provide the expected data type so that it will be useful while debugging.

Reply:
No need, as Raz mentioned I will just remove those lines. It's for recursion and for iterating over underlaying iterable objects. If the object is not iterable I don't care of it cause I won't look for any image there.

Reply:
I don't see this was fixed but not a blocker for this PR